
From: Frank Hirtz <fhirtz@redhat.com>

I'd like to request that the following patch be included in the current
build tree.  It modifies the display of CommitAvail in meminfo to allow for
negative values.  In this instance, negative values are appropriate if one
does not have strict overcommit accounting enabled and are actively
overcommitting memory.

Signed-off-by: <fhirtz@redhat.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/proc/proc_misc.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN fs/proc/proc_misc.c~fix-meminfo-commitavail-to-allow-for-negative-values fs/proc/proc_misc.c
--- 25/fs/proc/proc_misc.c~fix-meminfo-commitavail-to-allow-for-negative-values	Wed Oct  6 16:11:18 2004
+++ 25-akpm/fs/proc/proc_misc.c	Wed Oct  6 16:11:18 2004
@@ -205,7 +205,7 @@ static int meminfo_read_proc(char *page,
 		"Slab:         %8lu kB\n"
 		"CommitLimit:  %8lu kB\n"
 		"Committed_AS: %8lu kB\n"
-		"CommitAvail:  %8lu kB\n"
+		"CommitAvail:  %8ld kB\n"
 		"PageTables:   %8lu kB\n"
 		"VmallocTotal: %8lu kB\n"
 		"VmallocUsed:  %8lu kB\n"
_
